static/js/protein/page.tsx

```diff
@@ -34,6 +48,19 @@ interface PageStateType {
   data: GraphNodes;
 }
 
+export interface ProteinVarType {
```

Comment:
for this interface and the next one, it looks like they're also exported in data_processing_utils.tsx, you don't need to export it in both files

Reply:
Done !